openedx/features/course_duration_limits/access.py

```diff
@@ -34,10 +34,7 @@ def __init__(self, user, course, expiration_date):
         error_code = "audit_expired"
         developer_message = "User {} had access to {} until {}".format(user, course, expiration_date)
         language = get_language()
-        if language and language.split('-')[0].lower() == 'es':
-            expiration_date = strftime_localized(expiration_date, '%-d de %b. de %Y').lower()
-        else:
-            expiration_date = strftime_localized(expiration_date, '%b. %-d, %Y')
+        expiration_date = strftime_localized(expiration_date, u'%b. %-d, %Y')
         user_message = _("Access expired on {expiration_date}").format(expiration_date=expiration_date)
```

Comment:
Looks like we need the fix here too? and a few lines down from here...

Reply:
yeah you're right.  
this was just the first commit of my main PR that put back the fix for the original issue, but it makes sense to add the fixes to all the other unicode strings into this PR from my main branch, which i've done